internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -143,7 +143,29 @@ func TestAccAdvancedCluster_basicTenant(t *testing.T) {
 				Config: configTenant(t, true, projectID, clusterNameUpdated, ""),
 				Check:  checkTenant(true, projectID, clusterNameUpdated),
 			},
-			acc.TestStepImportCluster(resourceName),
+		},
+	})
+}
+
+func TestAccAdvancedCluster_basicTenant_flexUpgrade(t *testing.T) {
```

Comment:
would consider if we want to remove basicTenant test as this new test would capture the same scenario